app/views/layouts/application.html.erb

```diff
@@ -4,36 +4,13 @@
     <title><%= meta_title %></title>
     <meta name="description" content="<%= meta_description %>">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <!-- Chatroom -->
-    <%= stylesheet_link_tag "chatroom" if controller_name == "chatroom" %>
-    <!-- Facebook Open Graph data -->
-    <meta property="og:title" content="<%= meta_title %>" />
-    <meta property="og:type" content="website" />
-    <meta property="og:url" content="<%= request.original_url %>" />
-    <meta property="og:image" content="<%= meta_image %>" />
-    <meta property="og:description" content="<%= meta_description %>" />
-    <meta property="og:site_name" content="<%= meta_title %>" />
-    <!-- Twitter Card data -->
-    <meta name="twitter:card" content="summary_large_image">
-    <meta name="twitter:site" content="<%= DEFAULT_META["twitter_account"] %>">
-    <meta name="twitter:title" content="<%= meta_title %>">
-    <meta name="twitter:description" content="<%= meta_description %>">
-    <meta name="twitter:creator" content="<%= DEFAULT_META["twitter_account"] %>">
-    <meta name="twitter:image:src" content="<%= meta_image %>">
     <!-- Favicon -->
     <%= favicon_link_tag(source = "favicon.png", options = {}) %>
     <!-- Meta tags -->
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
-    <!-- Google Fonts -->
-    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;700&family=Share+Tech+Mono&display=swap" rel="stylesheet">
-    <!-- Fontawesome -->
-    <script src="https://kit.fontawesome.com/2e7849f35e.js" crossorigin="anonymous"></script>
-    <!-- Algolia -->
-    <script src="https://cdn.jsdelivr.net/npm/places.js@1.19.0/dist/cdn/places.min.js"></script>
```

Comment:
Deprecated. Algolia is stopping Places project on March 2022. 
Another solution for form auto completion is needed. Probably through Mapbox Geocoder.
Will address this is the future.